firmware/console/binary_log/log_field.h

```diff
@@ -8,10 +8,10 @@ struct Writer;
 class LogField {
 public:
 	template <typename TValue, int TMult = 1>
-	LogField(const scaled_channel<TValue, TMult>& toRead, const char* name, const char* units, int8_t digits)
+	constexpr LogField(const scaled_channel<TValue, TMult>& toRead, const char* name, const char* units, int8_t digits)
 		: m_type(resolveType<TValue>())
 		, m_multiplier(TMult)
-		, m_addr(reinterpret_cast<const char*>(&toRead))
```

Comment:
this `reinterpret_cast` was the problem - all type punning is forbidden in `constexpr`